src/main/java/core/basesyntax/service/impl/FileReaderServiceImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileReaderService;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FileReaderServiceImpl implements FileReaderService {
+    private static final int OFFSET = 1;

```

Comment:
move line skipping to the parser
we should be able to use this service for reading any other files as well